journeyman/webdev/markdown.md

```diff
@@ -0,0 +1,26 @@
+# Markdown 
+
+## Intro
+
+Markdown is a text-to-html tool for web developers. It allows you to write information that is easy write, and easy to write than it converts it to an html structure for you.
+
+## Suggested Learning
+
+- Read the Intro and Syntax on [Markdown](https://daringfireball.net/projects/markdown/)
+- Watch [this video](https://www.youtube.com/watch?v=_gknWWa2OF0) to see more clearly how it is written and used.
+- View the different .md files in this Webdev repository
+
+## Requirements
+
+- Edit/add a README.md file in your Apprentice Project
```

Comment:
Since you don't have to have completed an apprentice project to do this badge, I would prefer if this was more generic. Perhaps you could have them write a markdown file separately, or have them use an online markdown editor, like [Dillinger](http://dillinger.io), [StackEdit](https://stackedit.io/editor), [or even this one!](https://jbt.github.io/markdown-editor/)